src/c-writer.cc

```diff
@@ -1142,11 +1142,11 @@ void CWriter::Write(const StackVar& sv) {
 const char* CWriter::GetCTypeName(const Type& type) {
   // clang-format off
   switch (type) {
-  case Type::I32: return "u32";
-  case Type::I64: return "u64";
-  case Type::F32: return "f32";
-  case Type::F64: return "f64";
-  case Type::V128: return "v128";
+  case Type::I32: return "uint32_t";
+  case Type::I64: return "uint64_t";
+  case Type::F32: return "float";
+  case Type::F64: return "double";
+  case Type::V128: return "simde_v128_t";
```

Comment:
Baking in a simde type seems mildly heavy-weight. My first preference would be to have a define just for this,  but probably not a big deal either way.

Reply:
Fair enough, we can leave this as simde_v128_t